arches_lingo/src/arches_lingo/pages/SchemeList.vue

```diff
@@ -1 +1,44 @@
-<template>Schemes list placeholder</template>
+<script setup lang="ts">
+import { onMounted, ref } from "vue";
+import SchemeCard from "@/arches_lingo/components/scheme/SchemeCard.vue";
+import { fetchSchemes } from "@/arches_lingo/api.ts";
+
+import type { SchemeResource } from "@/arches_lingo/types";
+
+const schemes = ref<SchemeResource[]>([]);
+
+onMounted(async () => {
+    schemes.value = await fetchSchemes();
+    schemes.value.unshift({
```

Comment:
This is an interesting pattern. An equally valid pattern would be to manually create a list item for this entry. However I like this just fine 👍